client/post-editor/controller.js

```diff
@@ -91,13 +92,21 @@ function maybeRedirect( context, postType, site ) {
 	return false;
 }
 
-function getPressThisContent( text, url, title ) {
-	return ReactDomServer.renderToStaticMarkup(
-		<p>
-			{ text ? <blockquote>{ text }</blockquote> : null }
-			via <a href={ url }>{ title }</a>.
-		</p>
-	);
+function getPressThisContent( text, url, title, image, embed ) {
+	let pieces = [];
```

Comment:
`const` provides us with a constant reference, not immutability. I was looking for a reference change here due to the `let` and was confused when I didn't find one.

If we really feel strongly on using `const` only for primitive values like strings, booleans and numbers, and `let` on class instances, objects, and arrays this should be updated in our coding guidelines.